core/os/android/adb/device.go

```diff
@@ -110,6 +111,17 @@ func Devices(ctx context.Context) (DeviceList, error) {
 	return out, nil
 }
 
+func gapidApkPackageName(abi *device.ABI) string {
```

Comment:
This is a dupe of `gapidapk.PackageName`

Reply:
I can't use gapidapk.PackageName, there's cycle reference

Reply:
I suppose, then, move the `LaunchPerfettoProducer` function into the gapidapk package.

Reply:
That doesn't solve the problem, we still need to call `LaunchPerfettoProducer` when we construct the device info for a newly connected device, right?